source/_patterns/00-base/_variables.scss

```diff
@@ -0,0 +1,37 @@
+///
```

Comment:
Are we going to aim to have this more fully documented on the front end docs project?

Reply:
yes. this is mostly a companion to the large comment in _base.scss, the other "main" sass file, to hopefully really underscore the printing/nonprinting difference. that distinction should/will be covered thoroughly in the FE docs